server/medical-contact/collection.ts

```diff
@@ -50,8 +50,8 @@ class MedicalContactCollection {
    * @return {Promise<HydratedDocument<MedicalContact>[]>} - An array of all of the medical contacts sorted in alphabetical order by last name
    */
   static async findAllByOwnerId(ownerId: string): Promise<Array<HydratedDocument<MedicalContact>>> {
-    //const owner = await AccountCollection.findOneByAccountId(ownerId);
-    return MedicalContactModel.find({ownerId: ownerId}).sort({last_name: 1});
+    // Const owner = await AccountCollection.findOneByAccountId(ownerId);
```

Comment:
why change the capitalization of const?

Reply:
No idea – but given how the app has been working fine, I'll just delete the comments anyways.